### ISSUE TITLE
CPS Counter is broken

### ISSUE THREAD

**Issue by OrangeMan-432** [PeytonPlayz595/Shadow-4.0#26]
The CPS counter works until around 6 CPS, and then as I click faster, it displays slower, and then over about 10 CPS, it shows 0. I get 12-14 CPS, but this shows 0 constantly and has since when I started using it around version 2.5. I attached a video of me clicking in game and on a CPS test.
[Screen recording 2024-07-07 11.51.36 PM.webm](https://github.com/PeytonPlayz595/Shadow-4.0/assets/167663529/05c65a55-925d-4300-bdb0-87a8b6f2f29d)

**Comment by PeytonPlayz595** [comment]
Yeah this is some code that Fatal wrote back in Shadow 2.5 and I've been planning to rewrite it for a while
